helloworld/util/backend.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from helloworld.util.mode import Mode, get_mode
+
+
+def get_backend_url(name: str) -> str:
+    if get_mode() == Mode.DEV:
+        from django.conf import settings
+
+        return f"http://localhost:{settings.DEV_PORTS[name]}"
```

Comment:
there is no need for DEV_PORTS to be in django settings.. I suggest moving them to a constants.py file (for example) and thus avoiding the inline import.

Reply:
Good idea, done.